dockers/docker-fpm-frr/zsocket.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+secs=30
```

Comment:
do we really need to wait this long?

Reply:
I had put max wait. It will not matter since the script will exit as soon as zebra socket is available.